features/step_definitions/lolcommits_steps.rb

```diff
@@ -73,7 +73,7 @@
 end
 
 Then /^I should be (prompted for|presented) "(.*?)"$/ do |_, prompt|
-  assert @stdout.read.to_s.include?(prompt)
+  expect(@stdout.read.to_s).to include(prompt)
```

Comment:
Yeah, I would have preferred that, but there is an 3-way incompatibility with minitest/cucumber/aruba because aruba depends on rspec-expectations (see 6bc799a), and none of the workarounds functioned in our case that I could get to work.  Thankfully it's only for the few assertions in the cuke tests.  From looking through the issues boards it sounds like this may be addressed by the next major version of cucumber.